tarsier-service/service/src/db.py

```diff
@@ -0,0 +1,80 @@
+import os
+from typing import Any
+from sqlalchemy import create_engine, text, Row
+import uuid
+import json
+
+host = os.environ.get("DB_HOST") or "localhost"
+engine = create_engine(f"mysql+pymysql://user:password@{host}:3306/db") #docker compose ...@db:3306/db"
```

Comment:
i dont think think we are doing engine.dispose on shutdown here. eg (from our prod app)

    """
    @app.on_event("shutdown")
    async def _shutdown() -> None:
        await app.state.db_primary_engine.dispose()
        await app.state.db_replica_engine.dispose()
        await shutdown_kafka(app)
"""